misc/specs/atrium-specs-common/src/main/kotlin/ch/tutteli/atrium/specs/integration/IterableHasNotNextAssertionsSpec.kt

```diff
@@ -0,0 +1,48 @@
+package ch.tutteli.atrium.specs.integration
+
+import ch.tutteli.atrium.api.fluent.en_GB.messageContains
+import ch.tutteli.atrium.api.fluent.en_GB.toThrow
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.specs.Fun0
+import ch.tutteli.atrium.specs.SubjectLessSpec
+import ch.tutteli.atrium.specs.describeFunTemplate
+import ch.tutteli.atrium.specs.forSubjectLess
+import ch.tutteli.atrium.specs.include
+import ch.tutteli.atrium.specs.lambda
+import ch.tutteli.atrium.specs.name
+import ch.tutteli.atrium.translations.DescriptionBasic
+import ch.tutteli.atrium.translations.DescriptionIterableAssertion
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.Suite
+
+abstract class IterableHasNotNextAssertionsSpec(
```

Comment:
I would fuse IterableHasNotNextAssertionsSpec with IterableHasNextAssertionsSpec (as we did for CollectionAssertionsSpec)